util/config/config.go

```diff
@@ -97,6 +99,10 @@ var (
 		0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00,
 		0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x00, 0x01,
 	}
+	MaxNameDuration = HeightDependentInt32{
```

Comment:
There is no need to use HeightDependentInt32 if it's not height dependent :)

Reply:
should we keep it so we can make duration value flexible?

Reply:
We can change it from const to HeightDependentInt32 anytime in the future. Only a few lines need to be changed.